src/scripts/import-open-api.ts

```diff
@@ -774,6 +915,7 @@ const importOpenApi = async ({
 
   resolveDiscriminator(specs);
 
+  let outputImports = `/* Generated by restful-react */\n\n${customImport ?? ""}`;
```

Comment:
The previous version was a bit better with the indentation:

```tsx
${customImport ? `\n${customImport}\n` : ""}
```

- No extra `\n` if we don't have any `customImport`
- A trailing `\n` to avoid `customImport` to be merge with the other imports